crates/runtime/src/logic.rs

```diff
@@ -142,6 +142,18 @@ impl<'a> VMLogic<'a> {
         }
         .build()
     }
+
+    /// Sends a blockchain proposal through the WASM bridge.
+    ///
+    /// # Parameters
+    ///
+    /// * `actions` - The actions that make up the proposal.
+    ///               TODO: This will be a `Vec<ProposalAction>` when available.
+    ///
+    pub fn send_proposal(&mut self, _actions: Vec<Vec<u8>>) -> VMLogicResult<()> {
+        // TODO: Implement actual blockchain call
+        Ok(())
+    }
```

Comment:
we shouldn't need a function here, `send_proposal` should follow the model of accommodations made for `send_action`

Reply:
Gotcha. I've added a `proposals` field and am now pushing it in the same way.